src/style.css

```diff
@@ -0,0 +1,33 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  box-sizing: border-box;
+  height: 96px;
+  width: 96px;
+}
```

Comment:
You already have box-sizing rule in * selector.

Also don't hardcode container height and width, let content do it. Actually you can remove container div

![image](https://github.com/mate-academy/layout_stars/assets/80700247/c0d34caf-24d2-4402-87b7-68194d2241ec)